File: CoWorkHub/CoWorkHub.Model/PaymentMethod.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CoWorkHub.Model
{
    public class PaymentMethod
    {
        public int PaymentMethodId { get; set; }
        public string PaymentMethodName { get; set; } = null!;
        public bool IsDeleted { get; set; } = false;
    }
}

File: CoWorkHub/UI/coworkhub_mobile/lib/models/payment_method.dart
import 'package:json_annotation/json_annotation.dart';

part 'payment_method.g.dart';

@JsonSerializable()
class PaymentMethod {
  final int paymentMethodId;
  final String paymentMethodName;
  final bool isDeleted;

  PaymentMethod({
    required this.paymentMethodId,
    required this.paymentMethodName,
    required this.isDeleted,
  });

  factory PaymentMethod.fromJson(Map<String, dynamic> json) {
    return PaymentMethod(
      paymentMethodId: json['paymentMethodId'],
      paymentMethodName: json['paymentMethodName'],
      isDeleted: json['isDeleted'] ?? false,
    );
  }
}
